src/admin/class-datalayers.php

```diff
@@ -539,31 +744,92 @@ function remove_required($row) {
 	}
 
 	/**
-	 * Get the source fields from the datalayer file.
+	 * Fetch the data from an external source URL.
+	 *
+	 * @return mixed
+	 */
+	public static function fetch_datalayer_url_data() {
+		$url      = self::$datalayer_url;
+		$response = wp_remote_get( $url );
+
+		if ( is_wp_error( $response ) ) {
+			return [];
```

Comment:
Moet er niet iets van een foutmelding worden gegeven als de URL niet benaderbaar is?